ffi/capi/bindings/dart/FixedDecimal.g.dart

```diff
@@ -37,7 +37,7 @@ final class FixedDecimal implements ffi.Finalizable {
   /// See the [Rust documentation for `FloatPrecision`](https://docs.rs/fixed_decimal/latest/fixed_decimal/enum.FloatPrecision.html) for more information.
   ///
   /// Throws [Error] on failure.
-  factory FixedDecimal.fromDoubleWithLowerMagnitude(double f, int magnitude) {
+  static FixedDecimal create_from_double_with_lower_magnitude(double f, int magnitude) {
```

Comment:
this is probably caused by Diplomat introducing the constructor attribute, so you need to annotate `icu_capi`

Reply:
Yes, we should go through and tag stringifiers, constructors, named constructors, and accessors

Reply:
ok so in other words I can't do a clean update of diplomat?